frontend/__tests__/Pages/NewMetadataType/NewMetadataType.test.jsx

```diff
@@ -116,7 +116,7 @@ describe('Provides a form to create metadata types', () => {
     await wait(() => inputValue(descriptionInput, 'This is a TeSt desCRIPTION. L0rem ipsum.'));
     // before submission, only the tags should be fetched
     expect(fetch.mock.calls.length).toEqual(1);
-    const submitButton = getByText(new RegExp('Submit'));
+    const submitButton = getByText(new RegExp('ubmit'));
```

Comment:
ubmit?

Reply:
This is racism, and I am not okey with it